src/scenes/game-scene.ts

```diff
@@ -53,28 +53,28 @@ const levelConfig: LevelConfigType = {
     {
       tiledObjectName: 'hedgehog',
       classFactory: Hedgehog,
-      maxSize: 10,
+      maxSize: 100,
       runChildUpdate: true,
       autoSpawn: true,
     },
     {
       tiledObjectName: 'bat',
       classFactory: Bat,
-      maxSize: 10,
+      maxSize: 100,
       runChildUpdate: true,
       autoSpawn: true,
     },
     {
       tiledObjectName: 'tomato',
       classFactory: Tomato,
-      maxSize: 10,
+      maxSize: 100,
       runChildUpdate: true,
       autoSpawn: true,
     },
     {
       tiledObjectName: 'coin',
       classFactory: Coin,
-      maxSize: 10,
+      maxSize: 1000,
```

Comment:
Is this supposed to be 100?

Reply:
its the max amount allowed in the scene... i was hoping we could add many coins per level

even better would be to spawn and despawn items (and mobs) when the player is close to the marker

currently everything spawns at once when the level loads

Reply:
i think there is ~150 coins in the level atm

Reply:
oooh I thought this was the size of the object 😕